assign_rights/tests.py

```diff
@@ -44,14 +44,28 @@ def add_rights_shells(count=5):
             determination_date=random_date(),
             note=random_string(),
             start_date=random_date(),
-            end_date=random_date(),
+            end_date=random.choice([None, random_date()]),
             start_date_period=None,
             end_date_period=random.randint(0, 10),
-            end_date_open=False,
+            end_date_open=random.choice(["True", "False"]),
```

Comment:
Should these really be strings?

Reply:
Nope. Thanks for catching that. Just pushed up a fix.